src/test/java/io/anserini/collection/FeverSentenceCollectionTest.java

```diff
@@ -0,0 +1,120 @@
+    super.setUp();
+    collectionPath = Paths.get("src/test/resources/sample_docs/fever/collection1");
+    collection = new FeverSentenceCollection(collectionPath);
+    Path segment = Paths.get("src/test/resources/sample_docs/fever/collection1/segment1.jsonl");
+
+    segmentPaths.add(segment);
+    segmentDocCounts.put(segment, 14);
+
+    totalSegments = 1;
+    totalDocs = 14;
+
+    // empty document, should be ingested but not indexed
+    expected.put("", Map.of("id", "", "content", "", "raw", ""));
+
+    // document with empty content, should be ingested but not indexed
+    expected.put("Domain_Range_Ratio_-LRB-DRR-RRB-", Map.of(
+            "id", "Domain_Range_Ratio_-LRB-DRR-RRB-",
+            "content", "",
+            "raw", ""
+    ));
+
+    // regular document 1, should be ingested and indexed
+    expected.put("Shohei_Otani_0", Map.of(
+            "id", "Shohei_Otani_0",
+            "content", "is a Japanese baseball pitcher , outfielder , and designated hitter who plays for the " +
+                    "Hokkaido Nippon-Ham Fighters of Nippon Professional Baseball 's -LRB- NPB -RRB- Pacific League .",
+            "raw", "is a Japanese baseball pitcher , outfielder , and designated hitter who plays for the " +
+                    "Hokkaido Nippon-Ham Fighters of Nippon Professional Baseball 's -LRB- NPB -RRB- Pacific League ."
+    ));
+    expected.put("Shohei_Otani_1", Map.of(
+            "id", "Shohei_Otani_1",
+            "content", "He was the first pick of the Fighters in the 2012 draft .",
+            "raw", "He was the first pick of the Fighters in the 2012 draft ."
+    ));
+    expected.put("Shohei_Otani_2", Map.of(
+            "id", "Shohei_Otani_2",
+            "content", "He officially recorded the fastest pitch by a Japanese pitcher and in NPB history at " +
+                    "165 km/h .",
+            "raw", "He officially recorded the fastest pitch by a Japanese pitcher and in NPB history at 165 " +
+                    "km/h ."
+    ));
+    expected.put("Shohei_Otani_3", Map.of("id", "Shohei_Otani_3", "content", "", "raw", ""));
+
+    // regular document 2, should be ingested and indexed
+    expected.put("Kelvin,_North_Dakota_0", Map.of(
+            "id", "Kelvin,_North_Dakota_0",
+            "content", "Kelvin is an unincorporated community in Rolette County , in the U.S. state of North " +
+                    "Dakota .",
+            "raw", "Kelvin is an unincorporated community in Rolette County , in the U.S. state of North " +
+                    "Dakota ."
+    ));
+    expected.put("Kelvin,_North_Dakota_1", Map.of("id", "Kelvin,_North_Dakota_1", "content", "", "raw", ""));
+
+    // regular document 3 with unicode characters and empty lines, should be ingested and indexed
+    expected.put("Styrkepr\u00f8ven_0", Map.of(
+            "id", "Styrkepr\u00f8ven_0",
+            "content", "Styrkepr\u00f8ven , also called Den Store Styrkepr\u00f8ven -LRB- The Great Trial of " +
```

Comment:
Should we use ftfy to fix encoding here? I think it works?

`fix_encoding("Styrkepr\u00f8ven")
`
Styrkeprøven

Reply:
I'm not 100% sure, but I thought `ftfy` was used to fix broken unicode? This is from their website:
> The word schön might appear as schÃ¶n

Strings in python3 are unicode-encoded by default, so I think we should be able to get desired results for free:
```
>>> print('Styrkepr\u00f8ven')
Styrkeprøven
```